### DISCUSSION TITLE
popup not showing

### DISCUSSION THREAD

**Discussion by rj10rd** [benzino77/tasmocompiler#250] [General]
after ctrl+c and then enter node server/server.js not working

**Comment by benzino77** [comment]
To tell the truth I'm just guessing what you are talking about....
To start the server on your own you have to run `node server/app.js` just like it is stated in the [README](https://github.com/benzino77/tasmocompiler#less-easy-way)

**Reply by rj10rd** [reply]
thanks but in https://github.com/benzino77/tasmocompiler/blob/master/ADVANCED.md
is node server/server.js

**Reply by benzino77** [reply]
So it has to be fixed. PRs are welcome.
